tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -107,12 +176,30 @@ def test_positive_update_rhev_name(self):
         5. Update the name of the created CR with valid string.
 
         @Assert: The rhev Compute Resource is updated
-
-        @Caseautomation: notautomated
         """
+        parameter_list = [
+                        ['URL', self.rhev_url, 'field'],
+                        ['Username', self.rhev_username, 'field'],
+                        ['Password', self.rhev_password, 'field'],
+                        ['Datacenter', self.rhev_datacenter, 'special select']
+                        ]
+        with Session(self.browser) as session:
+            newname = gen_string('alpha')
+            with self.subTest(newname):
+                name = gen_string('alpha')
+                make_resource(
+                    session,
+                    name=name,
+                    provider_type=FOREMAN_PROVIDERS['rhev'],
+                    parameter_list=parameter_list
+                )
+                search = self.compute_resource.search(name)
+                self.assertIsNotNone(search)
+                self.compute_resource.update(name=name, newname=newname)
+                search = self.compute_resource.search(newname)
```

Comment:
Don't need to use a variable here, make an assertion right away
`self.assertIsNotNone(self.compute_resource.search(name))`